pkg/minikube/logs/logs.go

```diff
@@ -243,6 +243,16 @@ func outputLastStart() error {
 	return nil
 }
 
+// OutputMinikubeLogs outputs the audit and last start logs only
+func OutputMinikubeLogs(lines int) {
```

Comment:
`logs.OutputMinikubeLogs` stutters and doesnt read well

how about, Offline or DisplayOffline() ?
```
// Offline outputs logs that doesn't need a running cluster.
func logs.Offline()


```

Reply:
That makes sense, I think having `Output` in the func name is important so there's no question whether the func returns or outputs the logs, and it also follows in convention with other funcs. I'll update with `logs.OutputOffline()`.

Reply:
`OutputOffline` sounds good